bin/cassandra

```diff
@@ -152,7 +152,7 @@ find_library()
     pattern=$1
     path=$(echo ${2} | tr ":" " ")
 
-    find $path -regex "$pattern" -print 2>/dev/null | head -n 1
+    find $path -regex "$pattern" -print 2>/dev/null | sort -r | head -n 1
```

Comment:
If we have more than one libjemalloc package installed, even by accident, like, both 1 and 2 versions, `sort -r` will sort it in descending order and head will take the first one so we always use the newest libjemalloc library.